content/background.html

```diff
@@ -265,7 +265,11 @@ <h3>CEGEP</h3>
   <span class="big">
   All students from Quebec attended CEGEP partially, meaning they did not graduate.
   </span>
-  Among respondents, no students from outside of Quebec attended CEGEP.
+  The Quebec schooling system requires students to finish CEGEP, a 2-year collegiate degree,
+  before applying to a Quebec university. This is in part due to secondary school finishing
```

Comment:
2-year post-secondary diploma, before enrolling in a Quebec university [because you apply in 2nd yea]

Reply:
from what i recall, isn't the CEGEP degree a diploma of collegiate studies (DEC)?

Reply:
DEC in French, DCS in English. Not a degree. But my proposed wording is acronym-free.